{
  "schema_version": "1.4.0",
  "id": "GHSA-2q4h-27m7-rj67",
  "modified": "2022-05-14T02:10:10Z",
  "published": "2022-05-14T02:10:10Z",
  "aliases": [
    "CVE-2013-2191"
  ],
  "details": "python-bugzilla before 0.9.0 does not validate X.509 certificates, which allows man-in-the-middle attackers to spoof Bugzilla servers via a crafted certificate.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2013-2191"
    },
    {
      "type": "WEB",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=951594"
    },
    {
      "type": "WEB",
      "url": "https://git.fedorahosted.org/cgit/python-bugzilla.git/commit/?id=a782282ee479ba4cc1b8b1d89700ac630ba83eef"
    },
    {
      "type": "WEB",
      "url": "https://lists.fedorahosted.org/pipermail/python-bugzilla/2013-June/000104.html"
    },
    {
      "type": "WEB",
      "url": "http://lists.opensuse.org/opensuse-updates/2013-07/msg00025.html"
    },
    {
      "type": "WEB",
      "url": "http://lists.opensuse.org/opensuse-updates/2013-07/msg00026.html"
    },
    {
      "type": "WEB",
      "url": "http://www.openwall.com/lists/oss-security/2013/06/19/6"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-20"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2014-02-08T00:55:00Z"
  }
}